src/SayMore/UI/ComponentEditors/ContributorsEditor.cs

```diff
@@ -234,16 +234,17 @@ private void SaveContributors()
 		{
 			string failureMessage;
 			_file.SetValue("contributions", _model.Contributions, out failureMessage);
-			_file.SetValue(SessionFileType.kParticipantsFieldName, GetParticipants(), out failureMessage);
+			_file.SetValue(SessionFileType.kParticipantsFieldName, GetParticipants(false), out failureMessage);
```

Comment:
This line is essentially a subset of the above information, right? I assume it is being stored for backwards compatibility. If so, probably needs a comment to that effect.